packages/plugin-typescript/README.md

```diff
@@ -99,11 +99,13 @@ If you would like to extend / override these options:
     };
     ```
 
-This will then process your JavaScript with TypeScript with the additional configuration settings you provide.  This also allows you to configure the rest of _tsconfig.json_ to support your IDE and local development environment settings.
+This will then process your JavaScript with TypeScript with the additional configuration settings you provide.  This also allows you to configure the rest of your _tsconfig.json_ to support your project specific IDE and local development environment settings.
 
-### Custom Pages
+### Pages
 
-By default, this plugin extends TypeScript support to processing SSR pages and API routes.  If you would like to _disable_ this, set the `servePage` option to `false`
+By default, this plugin extends TypeScript support for processing [SSR pages](https://www.greenwoodjs.dev/docs/pages/server-rendering/) and [API routes](https://www.greenwoodjs.dev/docs/pages/api-routes/).  For this feature, you will need to enable [custom imports](https://www.greenwoodjs.dev/docs/pages/server-rendering/#custom-imports).
```

Comment:
Hmmm... the formatting looks fine when view the file contents directly?  🤷‍♂️ 
https://github.com/ProjectEvergreen/greenwood/blob/53c324a94ae7e516253ee4139554382b93258ab8/packages/plugin-typescript/README.md
![Screenshot 2024-11-05 at 8 33 45 PM](https://github.com/user-attachments/assets/b2520a19-5bba-4e0a-b2f9-a4e039e92d56)